docs/tasks/configure-pod-container/configmap.md

```diff
@@ -107,7 +107,7 @@ metadata:
 
 ### Create ConfigMaps from files
 
-You can use `kubectl create configmap` to create a ConfigMap from an individual file, or from multiple files.
+You can use `kubectl create configmap` to create a ConfigMap from an individual file, multiple files or from an env-file.
```

Comment:
better add some texts explaining what a env-file should look like, for example, it contains "key=value" pairs and spaces before the equal sign is not allowed...

Reply:
yep, makes sense, added syntax rules for --env-files and converted output to -o for yaml.